course_discovery/apps/course_metadata/management/commands/refresh_course_metadata.py

```
@@ -94,9 +94,12 @@ def handle(self, *args, **options):
                     partner.oidc_secret,
                     token_type=token_type
                 )
-            except Exception:
-                logger.exception('No access token acquired through client_credential flow.')
-                raise
+            except Exception:  # pylint: disable=broad-except
+                message = '[{name}] Failed to retrieve access token through client_credential flow.'.format(
+                    name=partner.name
+                )
+                logger.exception(message)
+                continue
```

Can you clarify why we're safe continuing? Is there a ticket to trigger an alert somewhere else to follow up on this exception?

---

This fix seems okay as a temporary solution but could lead to a larger issue of these exceptions being ignored. Looks okay to me if there's a plan to address the exceptions with a different alerting process.

---

There is no Alert right now.  In the ticket I added that we need to add one, but the fix here will continue to process the connections to our other partners when a connection fails to work.  This way we will still collect data when one connection is not working correctly.